models.py

```diff
@@ -308,6 +307,8 @@ def raw(self, Data, output="Average"):
                 return np.var(out,axis=1)
             elif output == 'Both':
                 return np.average(out,axis=1), np.var(out,axis=1)
+            else:
+                return np.average(out, axis = 1), np.var(out, axis = 1)
```

Comment:
in runSampling in querier.py, batch2dict is called with query_function=self.config.al.query_mode = 'learned' by default, which is not taken into account by current raw in models.py